woocommerce/woocommerce
# Add Business services card

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:
This PR adds the new Business card to the in app-marketplace and renders the business card for the places where business card products would be shown. 





Closes https://github.com/Automattic/woocommerce.com/issues/19974.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Switch the WCCOM branch to the one in this PR https://github.com/Automattic/woocommerce.com/pull/20361 and rebuild the assets.
2. run `npm run wp:cli cache flush` on WCCOM branch for the extension API's to have the new fields that is required for business services card
3.  Make sure you have some business services product on your local WCCOM. you can confirm by going to https://woocommerce.test/wp-json/wccom-extensions/1.0/search?category=business-services&country=US and confirm response has some product, otherwise add `Business services` products on your local WCCOM
4. Now go back to Core branch  and In plugins/woocommerce-admin/client/marketplace/components/constants.ts, edit the MARKETPLACE_HOST const to be https://woocommerce.test
5. Go to /plugins/woocommerce and run pnpm --filter='@woocommerce/plugin-woocommerce' build
6. Go to `Business services` [tab](http://localhost:8888/wp-admin/admin.php?page=wc-admin&tab=business-services&path=%2Fextensions) 
7. confirm the business services card shown according to Figma [design](https://www.figma.com/file/5jyiYtoTF2ZUxgkulPVpOh/Business-Services?node-id=3-3406&mode=dev) for desktop and mobile
8. Test around this Issue
![Screenshot 2024-05-06 at 6 23 49 PM](https://github.com/woocommerce/woocommerce/assets/11981912/8aba4cdb-e3f9-40b6-bdd9-889a5e60542c)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
